packages/augur-core/source/contracts/trading/OICash.sol

```diff
@@ -0,0 +1,77 @@
+pragma solidity 0.5.4;
+
+import 'ROOT/IAugur.sol';
+import 'ROOT/libraries/Initializable.sol';
+import 'ROOT/libraries/token/VariableSupplyToken.sol';
+
+
+/**
+ * @title OI Cash
+ * @dev A Wrapper contract for the deployed Cash contract which Augur considers OI. Cash can be deposited and will count toward OI for reporting fee calculations and will extract a reporting fee on withdrawl
+ */
+contract OICash is VariableSupplyToken, Initializable {
```

Comment:
Shouldn't this be declared as IOICash? Otherwise, IOICash doesn't seem to be getting used.

Reply:
Never mind, it is getting used in OICashFactory.

Reply:
Should still be IOICash though, will add